File: generated-sources/php-ze-ph/mojang-api/src/App/com/github/asyncmc/mojang/api/php/ze/ph/model/UploadSkinRequest.php
<?php

namespace com.github.asyncmc.mojang.api.php.ze.ph.server\com.github.asyncmc.mojang.api.php.ze.ph.model;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class UploadSkinRequest
{
    /**
     * @DTA\Data(field="model", nullable=true)
     * @DTA\Strategy(name="Object", options={"type":\com.github.asyncmc.mojang.api.php.ze.ph.server\com.github.asyncmc.mojang.api.php.ze.ph.model\SkinModel::class})
     * @DTA\Validator(name="Dictionary", options={"type":\com.github.asyncmc.mojang.api.php.ze.ph.server\com.github.asyncmc.mojang.api.php.ze.ph.model\SkinModel::class})
     * @var \com.github.asyncmc.mojang.api.php.ze.ph.server\com.github.asyncmc.mojang.api.php.ze.ph.model\SkinModel
     */
    public $model;
    /**
     * The skin image in PNG format
     * @DTA\Data(field="file")
     * @DTA\Strategy(name="Object", options={"type":\SplFileObject::class})
     * @DTA\Validator(name="Dictionary", options={"type":\SplFileObject::class})
     * @var \SplFileObject
     */
    public $file;
}

File: generated-sources/kotlin-spring/mojang-api/src/main/kotlin/com/github/asyncmc/mojang/api/kotlin/spring/model/UploadSkinRequest.kt
package com.github.asyncmc.mojang.api.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import com.github.asyncmc.mojang.api.kotlin.spring.model.SkinModel
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param model 
 * @param file The skin image in PNG format
 */
data class UploadSkinRequest (

        @get:NotNull 
        @JsonProperty("file") val file: java.io.File,

        @JsonProperty("model") val model: SkinModel? = null
) {

}

